### PR TITLE
Fixes #1 - Make iframe of dynamic map responsive

### LINKED ISSUE #1: Make iframe of dynamic map responsive
I'm wondering with this module how easy it would be to embed some responsive CSS for the iframe.  I didn't really have much luck with the width and height as it didn't seem to recognise vw as a unit, pixels are no good for responsive and percent, while it works for width is problematic for the height.

I applied this CSS (Source: https://www.w3schools.com/howto/howto_css_responsive_iframes.asp) to the containing div and the iframe:

```css
.node-location .field-item {
  position: relative;
  overflow: hidden;
  width: 100%;
  padding-top: 56.25%; /* 16:9 Aspect Ratio (divide 9 by 16 = 0.5625) */
}

.node-location iframe {
  position: absolute;
  top: 0;
  left: 0;
  bottom: 0;
  right: 0;
  width: 100%;
  height: 100%;
}
```

I'm happy to add this to my site CSS, but for others, it may be better if we can integrate this.  I'm not sure of the best way to add CSS to a module like this, but I'm happy to have a go with a bit of a steer.

### PR DESCRIPTION
**Pull request by yorkshire-pudding**
Fixes #1 
As per #1 this:

- adds a containing div (with class) around the iframe (with added class) in template
- removes width and height from iframe as these are now covered by CSS
- adds a CSS file with classes for the div and iframe
- moves the template to templates folder
- loads the CSS as part of the module using `#attach`
- Updates the Readme to reflect these changes

I have tested with the old template file in place and it works in the previous manner, taking width and height.  However, as we've moved the template, if the old template remains in the parent folder, it will be ignored.  I'm not sure if that matters if they've overridden in the theme as presumably the theme would hook in any changes and apply to the new template.

### CODE REVIEW

File: templates/simple-gmap-output.tpl.php [hunk 1]
```diff
@@ -48,4 +53,4 @@
 ?>
 <p class="simple-gmap-address"><?php print $address_text; ?></p>
 <?php
-}
+}
```
Comment (jenlampton) [inline templates/simple-gmap-output.tpl.php]: Can you add an extra newline at the end of this file please?

Comment (yorkshire-pudding) [reply templates/simple-gmap-output.tpl.php]: Done

File: simple_gmap.info [hunk 2]
```diff
@@ -2,4 +2,4 @@ name = Simple Google Maps
 description = Provides a Google Maps link/map formatter for simple Text fields
 backdrop = 1.x
 type = module
-package = Fields
+package = Fields
```
Comment (jenlampton) [inline simple_gmap.info]: same thing here, we need another linebreak.

File: simple_gmap.module [hunk 3]
```diff
@@ -103,28 +103,35 @@ function simple_gmap_field_formatter_view($entity_type, $entity, $field, $instan
   if (!in_array($map_type, array('m', 'k', 'h', 'p'))) {
     $map_type = 'm';
   }
-
+
```
Comment (jenlampton) [inline simple_gmap.module]: looks like this might add some whitespace, we should remove the spaces.

File: css/simple_gmap.css [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+/* iframe for embedded map */
+.simple-gmap-iframe {
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  width: 100%;
+  height: 100%;
+}
```
Comment (jenlampton) [inline css/simple_gmap.css]: need an extra linebreak here too.

File: simple_gmap.info (hunk 2 above)
Comment (yorkshire-pudding) [reply simple_gmap.info]: Done

File: simple_gmap.module (hunk 3 above)
Comment (yorkshire-pudding) [reply simple_gmap.module]: Done

File: css/simple_gmap.css (hunk 4 above)
Comment (yorkshire-pudding) [reply css/simple_gmap.css]: Done

**Pull request merged**